flink-tests/src/test/java/org/apache/flink/test/checkpointing/ContinuousFileProcessingCheckpointITCase.java

```diff
@@ -77,9 +77,9 @@
 	@Before
 	public void createHDFS() throws IOException {
 		if (failoverStrategy.equals(FailoverStrategy.RestartPipelinedRegionStrategy)) {
-			// TODO the 'NO_OF_RETRIES' is useless for current RestartPipelinedRegionStrategy,
-			// for this ContinuousFileProcessingCheckpointITCase, using RestartPipelinedRegionStrategy would result in endless running.
-			throw new AssumptionViolatedException("ignored ContinuousFileProcessingCheckpointITCase when using RestartPipelinedRegionStrategy");
+			// TODO the 'NO_OF_RETRIES' is useless for current RestartPipelinedRegionFailoverStrategy,
```

Comment:
This is actually not `flip. RestartPipelinedRegionStrategy ` but `StreamFaultToleranceTestBase#FailoverStrategy`.
I think we need to rename the enums in it as well.

Reply:
I can't agree with you more. I will rename the enums

Reply:
@zhuzhurk Hi zhuzhu, Appreciation of your kind guidance , and I've updated it according to your comments